src/components/Popover/components/PopoverOverlay/PopoverOverlay.tsx

```diff
@@ -61,12 +62,7 @@ export default class PopoverOverlay extends React.PureComponent<Props, never> {
   render() {
     const {active} = this.props;
     return (
-      <Transition
-        in={active}
-        timeout={Duration.Fast}
-        mountOnEnter
-        unmountOnExit
-      >
+      <Transition in={active} timeout={durationFast} mountOnEnter unmountOnExit>
```

Comment:
Pretty certain this one was wrong too. Should be like the CSS which is base. Can you double check that I am correct here?